Tests/SentryTests/SentryOptionsTest.m

```diff
@@ -275,6 +275,23 @@ - (void)testDefaultBeforeBreadcrumb
     XCTAssertNil(options.beforeBreadcrumb);
 }
 
+- (void)testTracePropagationTargets
```

Comment:
`m`: Let's please also validate the default value. We only need to add a line somewhere here https://github.com/getsentry/sentry-cocoa/blob/5ecf9f6f92ee2598fd21b95d1a3af356d9969c50/Tests/SentryTests/SentryOptionsTest.m#L519